core/vm/environment.go

```diff
@@ -18,10 +18,21 @@ package vm
 
 import (
 	"math/big"
+	"reflect"
+	"errors"
 
 	"github.com/ethereumproject/go-ethereum/common"
 )
 
+// Type is the VM type
+type Type byte
+
+const (
+	ClassicVmTy Type = iota // Default standard VM
+	SputnikVmTy
+	MaxVmTy
```

Comment:
What is MaxVM?

Reply:
I think it's the count of possible VMs. -) It was in original code and  it's logically correct to have this constant.

Reply:
It looks like an unfinished idea/placeholder to me -- https://github.com/whilei/go-ethereum/commit/c5bd32b0ad1a3d0fd20a3d1014cc8a97d889dc28